tests/scalers/loki/loki_test.go

```diff
@@ -0,0 +1,262 @@
+//go:build e2e
+// +build e2e
+
+package loki_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/joho/godotenv"
+	. "github.com/kedacore/keda/v2/tests/helper"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/client-go/kubernetes"
+)
+
+// Load environment variables from .env file
+var _ = godotenv.Load("../../.env")
+
+const (
+	testName = "loki-test"
+)
+
+var (
+	testNamespace         = fmt.Sprintf("%s-ns", testName)
+	deploymentName        = fmt.Sprintf("%s-deployment", testName)
+	monitoredAppName      = fmt.Sprintf("%s-monitored-app", testName)
+	publishDeploymentName = fmt.Sprintf("%s-publish", testName)
+	scaledObjectName      = fmt.Sprintf("%s-so", testName)
+	lokiServerName        = fmt.Sprintf("%s-server", testName)
+	minReplicaCount       = 0
+	maxReplicaCount       = 3
+)
+
+type templateData struct {
+	TestNamespace         string
+	DeploymentName        string
+	MonitoredAppName      string
+	PublishDeploymentName string
+	ScaledObjectName      string
+	LokiServerName        string
+	MinReplicaCount       int
+	MaxReplicaCount       int
+}
+
+const (
+	deploymentTemplate = `apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: test-app
+  name: {{.DeploymentName}}
+  namespace: {{.TestNamespace}}
+spec:
+  replicas: 0
+  selector:
+    matchLabels:
+      app: test-app
+  template:
+    metadata:
+      labels:
+        app: test-app
+        type: keda-testing
+    spec:
+      containers:
+      - name: prom-test-app
+        image: quay.io/zroubalik/prometheus-app:latest
```

Comment:
do we need to use this image for any reason? I mean, can we just use a nginx image or similar? Are we using any special feature from this image? 
I ask because using that image we are attached to that image architecture and we can't execute this e2e test in ARM64 eg